dev/CommonStyles/ScrollViewer_themeresources.xaml

```diff
@@ -15,6 +15,13 @@
     </ResourceDictionary.ThemeDictionaries>
 
     <Thickness x:Key="ScrollViewerScrollBarMargin">1</Thickness>
+    
+    <x:String x:Key="ScrollViewerSeparatorExpandBeginTime">00:00:00.40</x:String>
+    <x:String x:Key="ScrollViewerSeparatorExpandDuration">00:00:00.1</x:String>
+    <x:String x:Key="ScrollViewerSeparatorContractBeginTime">00:00:02.00</x:String>
+    <x:String x:Key="ScrollViewerSeparatorContractDelay">00:00:02</x:String>
+    <x:String x:Key="ScrollViewerSeparatorContractDuration">00:00:00.1</x:String>
+    <x:String x:Key="ScrollViewerSeparatorContractFinalKeyframe">00:00:02.1</x:String>
```

Comment:
I'm not quite sure why these are necessary (or more specifically why they weren't *already* necessary), but these are resources present in the v1 XAML file, not present in the v2 XAML file, but referenced in both files, which was leading to a resource lookup issue that was resolved by putting them here as well.

Reply:
I don't understand either. I suspect these resources were found in the OS Xaml generic.xaml where they are defined too.
But why wouldn't they be found anymore? idk.